tests/vm_test.go

```diff
@@ -169,7 +170,30 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		type vmiBuilder func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume)
 
 		newVirtualMachineInstanceWithFileDisk := func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
-			return tests.NewRandomVirtualMachineInstanceWithFileDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), testsuite.GetTestNamespace(nil), corev1.ReadWriteOnce)
+			sc, foundSC := libstorage.GetRWOFileSystemStorageClass()
+			if !foundSC {
+				Skip("Skip test when Filesystem storage is not present")
```

Comment:
I don't mind doing this, but we need to be consistent in all tests, right now the method you suggested is not used in most cases.
We can create a new issue/PR to switch all skips, but I believe doing so in a single PR is wrong.

Reply:
I care a lot about consistency, but should not accept these in-code hidden Skips. I think that they should be removed one at a time. Please try to avoid them now. If you fail to do so here we can reconsider filing an issue.

Reply:
https://github.com/kubevirt/kubevirt/issues/11632

Reply:
Is the naked Expect `Expect(foundSC)` a common practice? I haven't seen this before

Anyway, I think this test would simply fail when ran against storage that doesn't support RWO Filesystem volumes.
Don't have an immediate example in mind but there's probably a permutation like this in the wilderness

Reply:
> Is the naked Expect Expect(foundSC) a common practice

not at all, I should have made a better effort with the suggested code.

Reply:
ah okay, just note it ended up in the commit and it passes CI